src/patternfly/components/Toolbar/examples/toolbar-expanded-example.hbs

```diff
@@ -26,7 +26,7 @@
       secondary
     {{/button}}
   {{/toolbar-action-group}}
-  {{#> dropdown id=(concat toolbar--id '-dropdown-simple') dropdown--IsBasicActionMenu="true" dropdown--IsExpanded="true" dropdown--modifier="pf-c-toolbar__action-list" dropdown--HasKebabIcon="true" dropdown-toggle--modifier="pf-m-plain"}}
+  {{#> dropdown id=(concat toolbar--id '-dropdown-simple') dropdown--IsBasicActionMenu="true" dropdown--IsExpanded="true" dropdown--modifier="pf-c-toolbar__action-list" dropdown--HasKebabIcon="true" dropdown-toggle--modifier="pf-m-plain" aria-label=(concat toolbar--id '-dropdown-simple')}}
```

Comment:
`aria-label` is the attribute we want for this element, but this attribute takes a text string and not an id. Only `aria-labelledby` and `aria-describedby` take `id` values of other elements so that they can use the text string that's defined for that other element. 

In this case, we can just change this:
` aria-label=(concat toolbar--id '-dropdown-simple')`
to be this instead:
`aria-label="Actions"`

Reply:
Replaced code with `aria-label="Actions"`.